### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by amnonskalka**
remove unnecessary\old functions.
fixing appending data to the main Data-frame.
add new query and new column to the data-frame.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -18,8 +18,7 @@
     conn = ut.oracle_connect(db_ip, db_user, db_pass, db_sid)
     sql = ut.read_query(sql_path)
     site_results = ut.collect_data(sql, conn, site)
-
-    print(site_results)
-
+    results = results.append(site_results)
```
Comment (shay-berman) [inline main.py:21]: why you set the results var here?

append func already adding the item to the list

File: utilities.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import pandas as pd
 import cx_Oracle as orcl
 
-result_columns = ['SITE', 'CLIENT_VERSION', 'VERSION_DATE', 'DB_HOST_NAME', 'ORACLE_VERSION','ORACLE_EDITION', 'INSTANCE_STATUS', 'RESTRICTED_LOGIN', 'DB_STATUS','DB_FREE_SPACE_GB']
+result_columns = ['SITE', 'CLIENT_VERSION', 'VERSION_DATE', 'DB_HOST_NAME', 'ORACLE_VERSION','ORACLE_EDITION', 'INSTANCE_STATUS', 'RESTRICTED_LOGIN', 'DB_STATUS', 'DB_FREE_SPACE_GB', 'DB_USED_SPACE_GB']
```
Comment (shay-berman) [inline utilities.py]: pep8

please break down each val to different line
so next time u add line it will be short diff

File: utilities.py [hunk 3]
```diff
@@ -28,29 +28,17 @@ def read_query(x):
...
-        run = curr.execute(i)
-        for result in run:
-            list_conv = list(result)
-            result_list += list_conv
-    result_list.insert(0, site)
-    return result_list
-
-
 def collect_data(sql, con, site):
+    '''the func is getting 3 parameters : sql - list of query's, con - connection string to connect the oracle, site - client site name
```
Comment (shay-berman) [inline utilities.py]: one line below
and eary param in diff line
pep8

**Review by shay-berman — COMMENTED** [review]
the content of the pr is not align with your pr description
please align

File: main.py (hunk 1 above)
Comment (amnonskalka) [reply main.py:21]: this is the syntax to add the SQL results to the main Data-Frame

**Pull request merged**